.gitignore

```diff
@@ -30,4 +30,7 @@ renderer-fuzz.zip
 workdir
 
 # IDE
-.idea
\ No newline at end of file
+.idea
+
+# Go modules
+go.sum
```

Comment:
Hi. `go.sum` shouldnt be in the `.gitignore`. And you need to add `go.sum` to the PR